__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const app = require("../db/app.js")
+const request = require("supertest")
+const db = require("../db/connection.js")
+const seed = require("../db/seeds/seed.js")
+const data = require("../db/data/test-data")
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+describe('GET /api/topics', () => {
+    test('1. Returns 200 status code', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+    test('2. Returns array of objects with correct properties', () => {
+        return request(app)
+        .get('/api/topics')
+        .then(({body}) => {
+            expect(body.topics).toHaveLength(3);
+            body.topics.forEach((topic) => {
+                expect(typeof(topic.slug)).toBe('string')
+                expect(typeof(topic.description)).toBe('string')
```

Comment:
nice while this works and there is no problem here.
Start taking a look into some jest matchers as they will be great going forward.
ToMatchObject and ObjectContaining are two that could be really helpful this week.